wait-for-it.yaml

```diff
@@ -0,0 +1,27 @@
+package:
+  name: wait-for-it
+  version: 0_git200823
+  epoch: 0
+  description: Pure bash script to test and wait on the availability of a TCP host and port
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+        - "*"
+      attestation: TODO
+      license: MIT
```

Comment:
Does this have a runtime dependency on bash too?

Reply:
yes it does, as the shell script requires some bash things.